oozie/oozie.sh

```diff
@@ -87,14 +79,48 @@ EOF
   tar -xvzf /usr/lib/oozie/oozie-sharelib.tar.gz
   ln -s share /usr/lib/oozie/share
   hadoop fs -mkdir -p /user/oozie/
-  hadoop fs -put share/ /user/oozie/
+  hadoop fs -put -f share/ /user/oozie/
+  local additional_nodes=$(/usr/share/google/get_metadata_value attributes/dataproc-master-additional | sed 's/,/\n/g' | wc -l)
+  if [[ ${additional_nodes} -ge 2 ]]; then
+    echo 'Starting configuration for HA'
+    local servers=$(cat /usr/lib/zookeeper/conf/zoo.cfg | grep 'server.' | sed 's/server.//g' | sed 's/:2888:3888//g' | cut -d'=' -f2- | sed 's/\n/,/g' | head -n 3 |sed 's/$/:2181,/g' | xargs -L3 | sed 's/.$//g')
```

Comment:
Please comment on what's happening here, particularly the sed commands. Also consider breaking it up into multiple lines :)